src/EventHandler/ConfigurableEventHandlerEntityBundle.php

```diff
@@ -16,6 +16,40 @@
 class ConfigurableEventHandlerEntityBundle extends ConfigurableEventHandlerBase {
 
   /**
+   * The bundles information for the entity.
+   *
+   * @var array
+   */
+  protected $bundlesInfo;
+
+  /**
+   * The entity info plugin definition.
+   *
+   * @var mixed
+   */
+  protected $entityInfo;
+
+  /**
+   * The entity type.
+   *
+   * @var string
+   */
+  protected $entityType;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->entityType = $this->getEventNameSuffix();
```

Comment:
hm, not easy. Else, I could see it being a ConfigurableEventTrait which has the helper. The classes that need it, can use the helper then. We should mention the trait from the configurable event interface then though, e.g. via @see